.github/workflows/hassfest.yaml

```
@@ -1,14 +1,36 @@
-name: Validate with hassfest
+name: Validate
 
 on:
+  workflow_dispatch:
+  schedule:
+    - cron:  "0 0 * * *"
```

YAMLlint (1.37.1)

[error] too many spaces after colon

(colons)